midealocal/device.py

```diff
@@ -201,41 +201,35 @@ def _authenticate_refresh_capabilities(self) -> None:
     def connect(self) -> bool:
         """Connect to device."""
         connected = False
-        for _ in range(3):
```

Comment:
I'm against an infinite loop here.

Reply:
The removal of the for loop is not infinite loop.